### PR TITLE
Reset general styling

### LINKED ISSUE #86: Section spacing and styling
Currently, the section spacing is spread in styles.css and also inside each component, giving unexpected spacing. There should be one clear way to do the section styling. 
It might be better to have generic padding/margin in styles.

Also there will be special styles on section that can override those - like full width cta sections, or light-blue background section with various size. Ideally, the components in them should not be aware of how the section is styled.

### PR DESCRIPTION
**Pull request by andreibogdan**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #86 

Test URLs:
- Before: https://main--bitdefender--hlxsites.hlx.page/solutions/
- After: https://86-section-spacing--bitdefender--hlxsites.hlx.page/solutions/

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -157,8 +158,18 @@ body.no-scroll {
   overflow: hidden;
 }
 
-main .section .default-content-wrapper { /* mobile */
-  padding: 0 var(--body-padding);
+.section {
```
Comment (rhudea) [inline styles/styles.css:161]: Let use `main .section` and just set the padding and max-width and adapt it to media queries.

File: styles/styles.css [hunk 2]
```diff
@@ -157,8 +158,18 @@ body.no-scroll {
...
+  width: 100%;
+  position: relative;
+  padding: 0;
+}
+
+.wrapper {
```
Comment (rhudea) [inline styles/styles.css:167]: Do we need this ? Seems that it should be enough to just set the padding, max-width and margin on section.
Are there any issues without adding `.wrapper`  ?

File: styles/styles.css [hunk 3]
```diff
@@ -181,19 +192,8 @@ header {
     margin: 0 auto;
   }
 
-  main .section .default-content-wrapper {
```
Comment (rhudea) [inline styles/styles.css:184]: I was thinking to just repurpose these, dropping the .default-content-wrapper.

Comment (andreibogdan) [reply styles/styles.css:184]: we will remove it after we will agree on the final structure

File: styles/styles.css [hunk 4]
```diff
@@ -186,17 +199,25 @@ header {
...
+
+@media (min-width: 1400px){
+  .section {
+    padding: 60px 0;
   }
 }
 
 @media (min-width: 1600px) { /* large desktop */
-  main .section .default-content-wrapper {
+  .section .default-content-wrapper {
```
Comment (rhudea) [inline styles/styles.css:218]: Please keep the `main .section` to remain consistent with how other rules are written wrt section

File: blocks/columns/columns.css [hunk 5]
```diff
@@ -38,15 +38,14 @@
 
   .columns-wrapper {
     max-width: var(--section-desktop-max-width);
-    padding: 0 var(--section-desktop-padding);
-    margin: 0 auto;
+    padding: 0 var(--section-large-desktop-padding);
```
Comment (rhudea) [inline blocks/columns/columns.css:41]: Are you sure ? Seems like the opposite - start with large and go to small as screen increases ?

File: styles/styles.css [hunk 6]
```diff
@@ -162,16 +163,28 @@ body.no-scroll {
...
-}
-
 header {
   height: var(--nav-height);
 }
 
+.section {
+  width: 100%;
+  position: relative;
+  padding: 32px 0;
```
Comment (rhudea) [inline styles/styles.css:173]: Please set a variable similar to width --section-mobile-padding and desktop padding

File: blocks/columns/columns.css (hunk 5 above)
Comment (andreibogdan) [reply blocks/columns/columns.css:41]: I am sure :) default in original site is 
```
padding: 0 20px;
max-width: 999pt;
```
and the value is changing to 50px between 992-1599
```
@media (max-width: 1599px)
.we-container {
    max-width: 975pt;
    padding: 0 50px;
}

```

Comment (rhudea) [reply blocks/columns/columns.css:41]: Then shouldn't the variable values be interchanged ? And why only in columns ? There are other components using the exact same style

File: styles/styles.css (hunk 6 above)
Comment (andreibogdan) [reply styles/styles.css]: done

File: styles/styles.css (hunk 4 above)
Comment (andreibogdan) [reply styles/styles.css]: done

File: styles/styles.css [hunk 7]
```diff
@@ -186,17 +199,25 @@ header {
...
-    max-width: var(--section-desktop-max-width);
-    padding: 0 var(--section-desktop-padding);
-    margin: 0 auto;
+    padding: 0 var(--section-large-desktop-padding);
+  }
+}
+
+@media (min-width: 1400px){
+  .section {
+    padding: 60px 0;
```
Comment (rhudea) [inline styles/styles.css:213]: To be consistent, this shouls use a variable and use `main .section`

File: blocks/columns/columns.css (hunk 5 above)
Comment (andreibogdan) [reply blocks/columns/columns.css]: I reverted css for columns

**Review by rhudea — APPROVED** [review]

**Pull request merged**